postgresql/config.go

```diff
@@ -255,9 +278,18 @@ func (c *Config) connStr(database string) string {
 		if c.featureSupported(featureFallbackApplicationName) {
 			connValues = append(connValues, quote(c.ApplicationName))
 		}
+		if c.SSLClientCert != nil {
+			connValues = append(
+				connValues,
+				quote(c.SSLClientCert.CertificatePath),
+				quote(c.SSLClientCert.KeyPath),
+				quote(c.SSLClientCert.RootKeyPath),
+			)
+		}
 		connStr = fmt.Sprintf(dsnFmt, connValues...)
 	}
 
+	fmt.Printf("connection string: `%s`", connStr)
```

Comment:
Could you remove it please.

Reply:
done!

Reply:
Ok, I've made the requested changes, thank you for your feedback!